OpenRA.Mods.Common/Traits/Sellable.cs

```diff
@@ -57,5 +62,15 @@ public void Sell(Actor self)
 			else
 				self.QueueActivity(new Sell());
 		}
+
+		public bool IsVisible(Player forPlayer)
+		{
+			return forPlayer == self.Owner;
```

Comment:
Since neither me nor obrakmann can reproduce it I will put this on hold until someone finds it again.